_src/custom-date-format.md

```diff
@@ -6,22 +6,22 @@ attribute is used to provide the logic for handling the custom representation.
 
 !PLAYGROUND 7185eb211a4822ce97184ae25fedda91
```

Comment:
This hash needs to be updated by pasting runnable code into play.rust-lang.org and clicking Share.

Reply:
I did not know that. I updated the Hash as requested. Maybe it would be useful to add this to CONTRIBUTING.md?

Reply:
Yes, good call. #160